js/index.js

```diff
@@ -1,100 +1,360 @@
+      }
+      /*onValidationError:function(key, data){
+        webix.message.position = "bottom";
+        webix.message({text:key.toUpperCase()+"  field is incorrect", type:"error"});
+      }*/
+    };
+  var list = {
+    type:"clean",
+    rows:[
+      {
+        height: 35,
+        view:"toolbar",
+        elements:[
+          {
+            view:"text",
+            id:"list_input",
+            on:{
+              onTimedKeyPress:function(){
+                var value = this.getValue().toLowerCase();
+                $$("list").filter(function(obj){
+                    return obj.name.toLowerCase().indexOf(value)==0;
+                })
+              }
+            },
+          },
+          {view:"button", id:"btn_sort_asc", value:"Sort asc", css:"webix_primary", width:120,
+            click:function(){
+              webix.message({text:"Sort asc", expire:350});
+              $$("list").sort("#name#","asc","string");
+            }
+          },
+          {view:"button", id:"btn_sort_desc", value:"Sort desc", css:"webix_primary", width:120,
+            click:function(){
+              webix.message({text:"Sort desc", expire:350});
+              $$("list").sort("#name#","desc","string");
+            }
+          }
+        ]
+      },
+      {
+        view:"list",
+        css:"user_list",
+        id:"list",
+        height:300,
+        scrollY:true,
+        scrollX:false,
+        select:true,
+        template:"#name# <span class='remove_list_item_btn webix_icon mdi mdi-close'></span>",
+        onClick:{
+          remove_list_item_btn:function(e, id){
+            this.remove(id);
+            return false;
+          }
+        },
+        url:users_data_link
+      }
+    ]
+  };
+  var chart = {
+    type:"clean",
+    rows:[
+      {
+        view:"chart",
+        type:"bar",
+        value:"#age#",
+        label:"#age#",
+        minHeight:300,
+        tooltip:"User: #name#",
+        border:true,
+        barWidth:40,
+        radius:1,
+        xAxis:{
+            template:"#age#"
+        },
+        gradient:"falling",
+        url:users_data_link
+      },
+      {
+        view:"label",
+        label: "<span class='gray_text'>Age</span>",
+        align:"center",
+        height:30
+      }
+    ]
+  };
+  var treetable = {
+    view:"treetable",
+    scrollY:true,
+    scrollX:false,
+    select:true,
+    columns:[
+      { id:"id", header:"", css:"treetable_id", width:50},
+      { id:"title",	header:"Title", css:"treetable_title", minWidth:300, template:"{common.treetable()} #title#"},
+      { id:"price",	header:"Price", minWidth:300, css:"treetable_price", fillspace:true}
+    ],
+    url:products_data_link
+  };
+  var main = {
+    cells:[
+    	{ id:"Dashboard", cols:[data,form]},
+      { id:"Users", rows:[list,chart]},
+      { id:"Products", rows:[treetable]},
+      { id:"Admin", template:"Admin View"}
+    ]
+  };
+  var bottombar = {
+    rows:[
+      { view:"template", height:0.1 },
```

Comment:
why do you need to use this small template?

Reply:
In "bottombar" I have used "label" as well as for bottom side of "Chart" I have used "Age label". There was no border between 2 labels. I have used  "{ view:"template", height:0.1 }" like a border between 2 labels. Was modified!